newdle/client/src/components/DurationPicker.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import {Dropdown} from 'semantic-ui-react';
+import styles from './DurationPicker.module.scss';
+
+function renderDurationOptions(min, max, interval) {
```

Comment:
I don't think `render` in the function name makes much sense here

Reply:
True. Would a name like `calculate` be more appropriate?

Reply:
not a fan of `calculate` unless math is involved. `getOptions` or `buildOptions` maybe?

Reply:
I'll go with `getDurationOptions` then. Sounds better indeed.